_episodes/17-preparation.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Preparing to Teach"
+block: "Building Teaching Skill"
```

Comment:
The diff for preparation.md is not visible here due to renaming. Most changes may be viewed in the diff on [this earlier PR in the sandbox repo](https://github.com/data-lessons/instructor-training/pull/38/files).